### ISSUE TITLE
Show git name instead of "You"

### ISSUE THREAD

**Issue by dannyxnda** [gitkraken/vscode-gitlens#1230]
Can I have a setting that makes Gitlens show my git user.name instead of "You"?
Thanks!

**Comment by eamodio** [comment]
There isn't a way to do that currently. Curiously, why do you want that?

**Comment by dan0705** [comment]
Yeah, Each time I change my git user.name, it will show the git user.name. So I just want to show the git user.name instead of "You" to make the same behavior.
It may be silly but thank you for answering. xD

**Comment by eamodio** [comment]
This isn't something that I will likely add myself, but if the community wants to champion it and provide a PR (after discussing) I'm certainly open to it -- of course, depending on the complexity and scope of the changes required.

**Comment by dan0705** [comment]
Thank you!

**Comment by r-o-m-i-t** [comment]
This would be very helpful.

**Comment by upsteer** [comment]
This would be very helpful.

**Comment by eamodio** [comment]
Closed by #5088

**Comment by nikolay-1986** [comment]
Verification: Passed

The setting `gitlens.defaultCurrentUserNameStyle` now provides the following display modes for the username:

"you":

<img width="2228" height="1335" alt="Image" src="https://github.com/user-attachments/assets/452c70a0-1df1-4dd3-aa2c-043b71219e7b" />

"name":
<img width="2235" height="1345" alt="Image" src="https://github.com/user-attachments/assets/d67e0d1e-451a-4eb6-9a76-9b12a2cb1649" />

"nameAndYou":

<img width="2238" height="1337" alt="Image" src="https://github.com/user-attachments/assets/cb72eb6d-7e2e-48b0-ab80-e50570da9abe" />
